packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "xdsl_meeting_slots": "Prise de rendez-vous",
+  "xdsl_meeting_error": "Une erreur est survenue lors la récupération des dates de rendez-vous disponibles : {{error}}",
+  "xdsl_meeting_description": "L'intervention d'un technicien est nécessaire afin de finaliser l'installation de votre ligne.",
+  "xdsl_meeting_programmed_meeting_select": "Veuillez choisir le créneau pour l'installation de celle-ci.",
+  "xdsl_meeting_programmed_meeting": "Un rendez-vous est sélectionné le <strong>{{day}}</strong> entre <strong>{{start}}</strong> et <strong>{{end}}</strong>.",
```

Comment:
You can avoid to have html tags by setting the `<strong>` by passing them directly in the html template.
(Here is an example: https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/product-offers.html#L59)